i18n/fr/building.md

```diff
@@ -82,135 +88,193 @@ deb [Enter your mirror here] testing main non-free contrib
+$ sudo ./b2 -d2 -d1 ${config_opts} --prefix=${BOOST_PREFIX} stage
+$ sudo ./b2 -d0 ${config_opts} --prefix=${BOOST_PREFIX} install
+```
+**Note : voir plus bas les instructions de compilation de OpenBSD**
+
+### Windows (MSYS2/MinGW-64)
+* Téléchargez l'[installateur MSYS2](http://msys2.github.io/), 64-bit ou 32-bit en fonction de votre système, et lancez-le.
+* Utilisez le raccourci associé à votre architecture pour lancer l'environnement MSYS2. Sur les systèmes 64-bit, ce devrait être le raccourci Shell MinGW-w64 Win64. Notez que si vous êtes sur Windows 64-bit, vous aurez les 2 environnements 64-bit et 32-bit.
+* Mettez à jour les paquets de votre installation MSYS2 :
+```bash
 pacman -Sy
 pacman -Su --ignoregroup base
 pacman -Su
 ```
-* For those of you already familiar with pacman, you can run the normal ```pacman -Syu``` to update, but you may get errors and need to restart MSYS2 if pacman's dependencies are updated.
-* Install dependencies: ```pacman -S make mingw-w64-x86_64-cmake mingw-w64-x86_64-gcc mingw-w64-x86_64-boost mingw-w64-x86_64-openssl```
-* Optional: ```mingw-w64-x86_64-doxygen```  (you'll need [Graphviz](http://graphviz.org/doc/winbuild.html) for doxygen)
-* Note: You'll need  ``` mingw-w64-x86_64-miniupnpc``` if you are behind a restrictive NAT firewall.
+* Pour ceux d'entre vous qui connaissent déjà pacman, vous pouvez lancer ```pacman -Syu``` pour mettre à jour, mais vous pourriez avoir des erreurs et avoir besoin de relancer MSYS2 si les dépendances de pacman sont mises à jour.
+* Installez les dépendances : ```pacman -S make mingw-w64-x86_64-cmake mingw-w64-x86_64-gcc mingw-w64-x86_64-boost mingw-w64-x86_64-openssl```
+* Facultatif : ```mingw-w64-x86_64-doxygen```  (vous aurez besoin de [Graphviz](http://graphviz.org/doc/winbuild.html) pour doxygen)
+* Note : Vous aurez besoin de ``` mingw-w64-x86_64-miniupnpc``` si vous êtes derrière un pare-feu NAT restrictif.
 
-## Step 3. Build   
+## Étape 3. Compilation
 
-### 1. Clone the repository
+### 1. Clonez le dépôt
 ```bash
 $ git clone --recursive https://github.com/monero-project/kovri
 ```
-### 2. Build Kovri and submodule dependencies with one command
+
+### 2. Compilez kovri et les dépendances *submodule* en une commande
 ```bash
-$ make # to decrease build-time, run make -j [available CPU cores]
+$ make  # pour réduire le temps de compilation : `make -j <available CPU cores>`. Par exemple : `make -j4`
 ```
-### 3. Install resources (configuration files + package resources)
+
+### 3. Installation
 ```bash
 $ make install
 ```
 
-- End-users MUST run ```make install``` for new installations
-- Developers SHOULD run ```make install``` after a fresh fetch
-
-### Other options you can use in place of step 2:
+- Les utilisateurs DOIVENT lancer `make install` pour les nouvelles installations
+- Les développeurs DEVRAIENT lancer `make install` après compilation
```

Comment:
> DOIVENT lancer
> DEVRAIENT lancer

:+1:

Reply:
sorry, should I make a change here?

Reply:
No. Sorry, I was just trying to highlight the importance of this nuance

Reply:
alright!